src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -2105,7 +2123,8 @@ internal static bool CheckValEscape(SyntaxNode node, BoundExpression expr, uint
                         checkingReceiver,
                         escapeFrom,
                         escapeTo,
-                        diagnostics);
+                        diagnostics,
+                        isRefEscape: false);
 
                 case BoundKind.ObjectCreationExpression:
```

Comment:
It just occurred to me, are we covering tuples?

Reply:
Tuples cannot contain ref-structs since they are themselves not ref structs